qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java

```diff
@@ -108,8 +107,7 @@ public void addVisitNote() {
 
 	@Then("the system adds the note into visit note table")
 	public void systemAddsVisitNote() {
-		assertEquals(DIAGNOSIS_PRIMARY, visitNotePage.primaryDiagnosis());
-		assertEquals(DIAGNOSIS_SECONDARY, visitNotePage.secondaryDiagnosis());
```

Comment:
Why are these tests removed?

Reply:
These tests are replaced by assert Texts because the answer is not working since its element keeps un stable, More explanation is resting down.

Reply:
> element keeps un stable, More explanation is resting down.

What is makes the tests unstable in github actions and yet they are passing well in CI?

Reply:
@sherrif10 can you also answer the above?

Reply:
It is probably the overloading of the server and hence some files not getting well triggered hence resulting into unstable.